crates/ruff_python_parser/src/semantic_errors.rs

```diff
@@ -105,6 +105,19 @@ impl SemanticSyntaxChecker {
                         *range,
                     );
                 }
+
+                // test_ok single_starred_assignment_value
+                // _ = 4
+                // _ = [4]
+                // _ = (*[1],)
+                // _ = *[1],
+
+                // test_err single_starred_assignment_value
```

Comment:
Same here: Let's use `assign_stmt_starred_expr_value` to have the "assign_stmt_" prefix.